Templates/SasToken-CSharp/function.json

```diff
@@ -0,0 +1,16 @@
+{
+    "disabled": false,
+    "bindings": [
+        {
+            "authLevel": "function",
+            "name": "req",
+            "type": "httpTrigger",
+            "direction": "in"
```

Comment:
What verbs do you support? If it's just POST, recommend specifying `"methods": [ "post" ]' here